.github/workflows/publish-image.yaml

```
@@ -0,0 +1,78 @@
+---
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+name: "Container: Image Build"
+on:
+  workflow_dispatch: {}
+  pull_request:
+    branches: ["main"]
```

```suggestion
    branches: ["coreweave"]
```

I think we want to swap this?